plugin/src/geolocation.ts

```diff
@@ -0,0 +1,52 @@
+import { PostMessageProps } from "@web/extensions/geolocation/core/types";
+
+import html from "../dist/web/geolocation/core/index.html?raw";
+
+const reearth = (globalThis as any).reearth;
+
+reearth.ui.show(html, { width: 44, height: 44 });
+
+reearth.on("message", ({ action, payload }: PostMessageProps) => {
+  if (action === "flyTo") {
+    if (
+      payload.currentLocation.latitude !== undefined &&
+      payload.currentLocation.longitude !== undefined &&
+      payload.currentLocation.altitude !== undefined
+    ) {
+      reearth.layers.add({
+        extensionId: "marker",
+        isVisible: true,
+        title: "myLocation",
+        property: {
+          default: {
+            location: {
+              lat: payload.currentLocation.latitude,
+              lng: payload.currentLocation.longitude,
+            },
+            style: "point",
+            pointColor: "#12BDE2",
+            pointOutlineWidth: 4,
+            pointOutlineColor: "#FFFFFF",
+          },
+        },
+      });
+
+      const initCameraPos = reearth.camera.position;
```

Comment:
It isn't a big deal, but you can clean up a little and do 
```
const cameraHeading = reearth.camera.position?.heading ?? 0;
```
And then just use cameraHeading in the flyTo.

By the way, is 0 an appropriate heading value?

Reply:
and for the values it looks good in the test , and Red has tested it before , but I will mention him again to confirm that

Reply:
fc8de4437577d111f90b0a1c7db09d37c48d4de6